pyrobosim/pyrobosim/planning/pddlstream/mappings.py

```diff
@@ -61,6 +69,7 @@ def get_stream_info():
         # Streams (that sample)
         "s-navpose": StreamInfo(eager=False),
         "s-motion": StreamInfo(eager=False),
+        "s-grasp": StreamInfo(eager=False),
```

Comment:
Sadly, I don't think we can eager sample these (I tried), because the generated grasps depend on the instantaneous pose of the object and robot.

Nav pose sampling should work with eager sampling though, since the locations don't (yet) move. And these changes work, so I'll do that.

Reply:
well, that seemed to decrease run time a bit. I wasn't timing it exactly, though.